README.md

```diff
@@ -61,9 +61,22 @@ On Ubuntu and Raspberry Pi OS you can get the libraries by running
 ```
 $ sudo apt-get install libssl-dev libcurl4-openssl-dev liblog4cplus-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-base-apps gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-tools
 ```
+### Setup desired log level:
+Set up the desired log level. The log levels currently available with `log4cplus` are:
+1. `TRACE`
+2. `DEBUG` 
+3. `INFO`   
+4. `WARN`   
+5. `ERROR`   
+6. `FATAL`   
+
+To set a log level, update the log level value [here](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/blob/master/samples/kvs_log_configuration#L1)
```

Comment:
Can you move to the `kvs_log_configuration` to outside samples directory (the root of the SDK basically). Along with this, can we also change this line to set DEFAULT_LOG_FILE_PATH to `../kvs_log_configuration`?

https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/blob/master/src/gstreamer/gstkvssink.cpp#L106